File: services/identity-service/src/main/java/br/com/deltaglobalbank/identity/features/apiClients/createApiClient/CreateApiClientResponse.java
package br.com.deltaglobalbank.identity.features.apiClients.createApiClient;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

import br.com.deltaglobalbank.identity.domain.role.RoleCode;

public record CreateApiClientResponse(
    UUID id,
    UUID tenantId,
    String name,
    String description,
    String status,
    List<RoleCode> roles,
    Instant createdAt
) {
}

File: services/identity-service/src/main/kotlin/br/com/deltaglobalbank/identity/features/apiClients/createApiClient/CreateApiClientResponse.kt
package br.com.deltaglobalbank.identity.features.apiClients.createApiClient

import br.com.deltaglobalbank.identity.domain.role.RoleCode
import java.time.Instant
import java.util.UUID

data class CreateApiClientResponse(
    val id: UUID,
    val tenantId: UUID,
    val name: String,
    val description: String?,
    val status: String,
    val roles: List<RoleCode>,
    val createdAt: Instant
)
